index.html

```diff
@@ -42,15 +42,15 @@
                     alt="Entrepreneurs summit logo"></a>
             <nav class="nav">
                 <button class="menu-button">
-                    <i class="fa-solid fa-bars menu-open-icon"></i>
-                    <i class="fa-solid fa-xmark menu-close-icon"></i>
+                    <i class="fa-solid fa-bars menu-icon menu-icon-open"></i>
+                    <i class="fa-solid fa-xmark menu-icon menu-icon-close hidden"></i>
                 </button>
```

Comment:
- [ ] Clicking on the mobile menu button does nothing for both the `home` page and the `about` page. Kindly fix this issue 👍 

![Animation](https://user-images.githubusercontent.com/50721479/196989398-7979286f-a9c5-4c13-a681-6bcfdaced8dc.gif)